src/compas_timber/ghpython/components/CT_Joint_Options_LButt/code.py

```diff
@@ -0,0 +1,13 @@
+from ghpythonlib.componentbase import executingcomponent as component
+
+
+from compas_timber.connections import LButtJoint
+from compas_timber.connections.joint import JointOptions
+
+
+class MyComponent(component):
+    def RunScript(self, Gap, Small_Beam_Butts):
```

Comment:
inconsistent naming here, either match `metadata.json` with `CamelCase` or use `snake_case` to follow python convention.